src/components/SettingsPane/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+	Box,
+	Card,
+	CardContent,
+	CardHeader,
+	Stack,
+	Typography,
+} from '@mui/material';
+import { color } from '@balena/design-tokens';
+
+export interface SettingsPaneProps {
+	title?: string | React.ReactNode;
+	children: React.ReactNode;
+	danger?: boolean;
+}
+
+export interface SettingsPaneItemProps {
+	title: string | React.ReactNode;
+	description?: string | React.ReactNode;
+	action?: React.ReactNode;
+}
+
+export const SettingsPane = ({
+	title,
+	children,
+	danger,
+}: SettingsPaneProps) => (
+	<Card>
+		{title && (
+			<CardHeader
+				title={title}
+				sx={{
+					'.MuiCardHeader-title': {
+						color: danger ? color.text.danger.value : 'inherit',
+					},
+				}}
+			/>
```

Comment:
So that we don't have to explicitly pass `inherit`
```suggestion
				{...(danger && { sx: {
					'.MuiCardHeader-title': {
						color: danger ? color.text.danger.value : 'inherit',
					},
				}})}
			/>
```